.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,7 +13,7 @@ Contributors guide: https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
 - [ ] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
 - [ ] tests and/or benchmarks are included
 - [ ] documentation is changed or added
-- [ ] commit message follows commit guidelines
+- [ ] commit message follows [commit guidelines](../CONTRIBUTING.md#commit-message-guidelines)
```

Comment:
>Confirming the ../ will link correctly when the PR is created

It won't link properly when you raise a PR, as the PR comes under https://github.com/nodejs/node/compare. You can try it with a draft PR (or by editing this PR's title, but none of the relative links work:

 - [../ commit guidelines](../CONTRIBUTING.md)
 - [./ commit guidelines](./CONTRIBUTING.md)
 - [commit guidelines](CONTRIBUTING.md)


Had a similar issue [here](https://github.com/nodejs/citgm/pull/349).